packages/frontend/src/page/components/opening-countdown/countdown-frame/index.tsx

```diff
@@ -26,7 +29,7 @@ export const CountdownFrame = ({
                 }}
                 weight="medium"
             >
-                {timeFrame}
+                {duration.as("seconds") >= 0 ? duration.format(format) : "00"}
```

Comment:
Oh so that explains it. The negative values were appearing because of a custom implementation of the countdown element. It was weird because I checked the UI lib and I saw the lower limit to 0 being enforced there. Is there a specific reason we're not using the UI element? Too much customization needed?

Reply:
Yes, I think that's the reason we used a custom component, the style it's pretty different.